libnullpay/Cargo.toml

```diff
@@ -6,7 +6,7 @@ build = "build.rs"
 
 [lib]
 name = "nullpay"
-crate-type = ["cdylib", "rlib"]
+crate-type = ["staticlib","cdylib", "rlib"]
```

Comment:
seems like staticlib should be android-specific target

Reply:
I don't know if rust supports the conditional crate-type declaration. Does it? 
There are two issues related to it though, https://github.com/rust-lang/cargo/pull/5367 and https://github.com/rust-lang/cargo/issues/4900 .